src/lib/structures/Logger.ts

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable @typescript-eslint/no-unused-vars, @typescript-eslint/no-empty-function */
```

Comment:
I think the logger would go among the managers maybe? Not in structures?

Reply:
I've been thinking about this. Manager is not quite it, maybe I would put it more in utils/logger